src/drivers/cookie.js

```diff
@@ -0,0 +1,38 @@
+var cookies = require('browser-cookies');
+
+var CookieStore = module.exports = function CookieStore () { };
+CookieStore.backendName = 'cookie';
+
+CookieStore.supported = function () {
+	var ret = false;
+	if (typeof document !== 'undefined') {
+		try {
+			// Create cookie
+			document.cookie = 'cookietest=1';
+			ret = document.cookie.indexOf('cookietest=') !== -1;
+			// Delete cookie
+			document.cookie = 'cookietest=1; expires=Thu, 01-Jan-1970 00:00:01 GMT';
```

Comment:
yeah, "technically" they work the same.  Ill update it so its easier to see what I am doing